drivers/gles3/storage/config.cpp

```diff
@@ -30,18 +30,13 @@
 
 #ifdef GLES3_ENABLED
 
-#include "config.h"
```

Comment:
The reordering of includes is weird here.

`config.h` should still be first.

Then relative includes.

The global ones (`core/`, etc.).

Not sure where `platform_gl.h` should go in that model, I guess last as it's equivalent to thirdparty/system header includes?

Reply:
Actually `platform_gl.h` is already included by `config.h`, so it's redundant here.